tests/manager/searchCustomer/managerCanSearchCustomerByPostalCode.spec.js

```diff
@@ -1,35 +1,31 @@
 import { test } from '@playwright/test';
 import { faker } from '@faker-js/faker';
+import { AddCustomerPage } from '../../../src/pages/manager/AddCustomerPage';
+import { CustomersListPage } from '../../../src/pages/manager/CustomersListPage';
 
-let firstName;
-let lastName;
-let postalCode; 
+let createdFirstName;
+let createdLastName;
+let createdPostalCode;
 
 test.beforeEach( async ({ page }) => {
-  /* 
-  Pre-conditons:
-  1. Open Add Customer page
-  2. Fill the First Name.  
-  3. Fill the Last Name.
-  4. Fill the Postal Code.
-  5. Click [Add Customer].
-  */
-
-  firstName = faker.person.firstName();
-  lastName = faker.person.lastName();
-  postalCode = faker.location.zipCode(); 
+  const addCustomerPage = new AddCustomerPage(page);
 
+  createdFirstName = faker.person.firstName();
+  createdLastName = faker.person.lastName();
+  createdPostalCode = faker.location.zipCode();
 
+  await addCustomerPage.open();
+  await addCustomerPage.fillFirstNameField(createdFirstName);
+  await addCustomerPage.fillLastNameField(createdLastName);
+  await addCustomerPage.fillPostCodeField(createdPostalCode);
+  await addCustomerPage.clickAddCustomerButton();
 });
 
 test('Assert manager can search customer by Postal Code', async ({ page }) => {
-/* 
-Test:
-1. Open Customers page
-2. Fill the postalCode to the search field
-3. Assert customer row is present in the table. 
-4. Assert no other rows is present in the table.
-*/
-
+  const customersListPage = new CustomersListPage(page);
 
-});
\ No newline at end of file
+  await customersListPage.open();
+  await customersListPage.fillCertainValueInSearchField(createdPostalCode);
```

Comment:
```suggestion
  await customersListPage.fillValueInSearchField(createdPostalCode);
```
this function can shortened